forc/src/ops/forc_run.rs

```diff
@@ -56,39 +55,34 @@ pub async fn run(command: RunCommand) -> Result<()> {
 
     if command.dry_run {
         println!("{:?}", tx);
-        Ok(())
+        Ok(vec![])
     } else {
         let node_url = match &manifest.network {
             Some(network) => &network.url,
             _ => &command.node_url,
         };
-        let child = try_send_tx(node_url, &tx, command.pretty_print).await?;
-        if command.kill_node {
-            if let Some(mut child) = child {
-                child.kill().await.expect("Node should be killed");
-            }
-        }
```

Comment:
Good spot! I guess the `Child` was left over from the days where `forc run` would spin up a node?

Reply:
Thanks! That would make sense!